js_modules/dagster-ui/packages/ui-core/src/sensors/SensorTargetList.tsx

```diff
@@ -35,3 +64,131 @@ export const SensorTargetList = ({
     </Box>
   );
 };
+
+const AssetSelectionLink = ({assetSelection}: {assetSelection: SensorAssetSelectionFragment}) => {
+  const [showAssetSelection, setShowAssetSelection] = React.useState(false);
+
+  const sortedAssets = React.useMemo(() => {
+    return assetSelection.assets
+      .slice()
+      .sort((a, b) =>
+        COMMON_COLLATOR.compare(displayNameForAssetKey(a.key), displayNameForAssetKey(b.key)),
+      );
+  }, [assetSelection.assets]);
+
+  const assetsWithAMP = React.useMemo(
+    () => sortedAssets.filter((asset) => !!asset.definition?.autoMaterializePolicy),
+    [sortedAssets],
+  );
+  const assetsWithoutAMP = React.useMemo(
+    () => sortedAssets.filter((asset) => !asset.definition?.autoMaterializePolicy),
+    [sortedAssets],
+  );
+
```

Comment:
I think this information is only relevant for automation policy sensors and we should only do this breakdown for sensors of that sensor type - kind of a non sequiter in other contexts

@braunjj @sryza thoughts on that?